payments/vipps/after-payment.md

```diff
@@ -664,16 +643,17 @@ Content-Type: application/json
 ```
 
 {:.table .table-striped}
-| Property       | Type   | Description                                                                                          |
-| :------------- | :----- | :--------------------------------------------------------------------------------------------------- |
-| payment        | string | The relative URI of the payment that the reversal transactions belong to.                            |
-| reversalList   | array  | The array of reversal transaction objects.                                                           |
-| reversalList[] | object | The reversal transaction object representation of the reversal transaction resource described below. |
+| Property                 | Type     | Description                                                                                          |
+| :----------------------- | :------- | :--------------------------------------------------------------------------------------------------- |
+| `payment`                | `string` | The relative URI of the payment that the reversal transactions belong to.                            |
+| `reversals`              | `object` | The reversal object.                                                                                 |
+| └➔&nbsp;`reversalList`   | `array`  | The array of reversal transaction objects.                                                           |
+| └➔&nbsp;`reversalList[]` | `object` | The reversal transaction object representation of the reversal transaction resource described below. |
 
 #### Create reversal transaction
 
-The `create-reversal` operation reverses a previously created and
-captured payment.
+A `reversal` transaction can be created after a completed authorization by
+sending a request to `/psp/vipps/payments/<payment-id>/reversals`.
```

Comment:
This sentence goes against the hypermedia principles of our APIs. A URI should be discovered by finding the `href` of an operation from a previous request. Please describe which `rel` one should look for to find the reversal operation instead of describing the URI to perform the request against.

Reply:
Fixfixed